src/middlewared/middlewared/plugins/chart_releases_linux/validation.py

```diff
@@ -136,3 +163,36 @@ async def validate_container_image(self, verrors, value, question, schema_name,
         else:
             if not digest:
                 verrors.add(schema_name, f'Unable to retrieve {tag!r} container image tag details.')
+
+    @private
+    async def validate_custom_host_path(self, verrors, path, question, schema_name, release_data):
+        if not path:
+            return
+
+        is_cluster = path.startswith(FuseConfig.FUSE_PATH_SUBST.value)
+        if is_cluster:
+            verrors.add(schema_name, f"Fuse mounted paths are not allowed: {path}.")
+
+        p = Path(path)
+        if not p.is_absolute():
+            verrors.add(schema_name, f"Must be an absolute path: {path}.")
+
+        if os.path.realpath(path).startswith("/root/.ssh"):
+            verrors.add(schema_name, f"Changes to permissions under /root/.ssh are not permitted: {path}.")
```

Comment:
This is not really required as the validation would enforce that the path is under `/mnt` ?